src/Resources/app/administration/src/component/adyen-payment-capture/adyen-payment-capture.html.twig

```diff
@@ -1,44 +1,91 @@
+                            {{ $tc('adyen.sendCaptureRequest') }}
+                        </sw-button>
+                    </template>
+                </sw-modal>
+            </div>
+            <sw-data-grid
+                v-if="captureRequests.length"
+                :dataSource="captureRequests"
+                :columns="columns"
+                :showSelection="false"
+                :showActions="false"
+                :isLoading="isLoading">
+            </sw-data-grid>
+        </div>
+        <div v-else>
+            {{ $tc('adyen.error') }}
         </div>
-        <sw-data-grid
-            v-if="captureRequests.length"
-            :dataSource="captureRequests"
-            :columns="columns"
-            :showSelection="false"
-            :showActions="false"
-            :isLoading="isLoading">
-        </sw-data-grid>
-    </div>
-    <div v-else>
-        {{ $tc('adyen.error') }}
-    </div>
-</sw-card>
+    </sw-card>
+{% endblock %}
+
+{#<sw-card :isLoading="isLoading" :title="$tc('adyen.adyenPaymentCaptureTitle')" v-if="showWidget" :large="false" :hero="false">#}
+{#    <div v-if="!errorOccurred">#}
+{#        <div class="adyen-card-header">#}
+{#            <div class="adyen-card-header-first">#}
+{#                <span v-if="!captureRequests.length">{{ $tc('adyen.noCaptureRequests') }}</span>#}
+{#            </div>#}
+{#            <div id="modalButtonContainer">#}
+{#                <sw-button#}
+{#                    :disabled="!allowCapture"#}
+{#                    :square="false"#}
+{#                    :block="false"#}
+{#                    @click="openModal"#}
+{#                    class="sw-button--small">#}
+{#                    {{ $tc('adyen.sendCaptureRequest') }}#}
+{#                </sw-button>#}
+{#            </div>#}
+{#            <sw-modal v-if="showModal"#}
+{#                    @modal-close="onCloseModal"#}
+{#                    :title="$tc('adyen.captureConfirm') + ': ' + order.orderNumber"#}
+{#                    variant="small">#}
+{#                <template #modal-footer>#}
+{#                    <sw-button @click="onCloseModal" size="small">#}
+{#                        {{ $tc('global.default.cancel') }}#}
+{#                    </sw-button>#}
+
+{#                    <sw-button @click="onSubmitCapture" variant="primary" size="small" :isLoading="isLoading">#}
+{#                        {{ $tc('adyen.sendCaptureRequest') }}#}
+{#                    </sw-button>#}
+{#                </template>#}
+{#            </sw-modal>#}
+{#        </div>#}
+{#        <sw-data-grid#}
+{#            v-if="captureRequests.length"#}
+{#            :dataSource="captureRequests"#}
+{#            :columns="columns"#}
+{#            :showSelection="false"#}
+{#            :showActions="false"#}
+{#            :isLoading="isLoading">#}
+{#        </sw-data-grid>#}
+{#    </div>#}
+{#    <div v-else>#}
+{#        {{ $tc('adyen.error') }}#}
```

Comment:
do we need commenting those lines? or we can just delete them?

Reply:
It's a leftover from development. I removed, thanks for the heads-up!